packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -135,13 +139,14 @@ export default function TdpClientCanvas(props: Props) {
 
 export type Props = {
   tdpCli: TdpClient | null;
-  tdpCliOnImageFragment: (
-    ctx: CanvasRenderingContext2D,
-    data: ImageFragment
-  ) => void;
+  tdpCliOnPngFrame: (ctx: CanvasRenderingContext2D, pngFrame: PngFrame) => void;
```

Comment:
Updating the naming to more closely correspond with [the TDP spec](https://github.com/gravitational/teleport/blob/master/rfd/0037-desktop-access-protocol.md).